packages/react-core/src/components/MenuToggle/MenuToggle.tsx

```diff
@@ -37,6 +37,8 @@ export class MenuToggleBase extends React.Component<MenuToggleProps> {
     const { children, className, icon, badge, isExpanded, isDisabled, variant, innerRef, ...props } = this.props;
 
     const isPlain = variant === 'plain';
+    const isPlainText = variant === 'plainText';
```

Comment:
you could combine these into one then line 58 would not have to change.
`const isPlain = variant === 'plain' || variant === 'plainText'`

Reply:
Then i'd have to change the logic on lines 71-72. which I could do. But it's not true that if `plaintext` then `plain`. Plaintext also uses the expand/collapse arrow, unlike plain.